test/src/xquery/arrays/arrays.xql

```diff
@@ -136,12 +136,28 @@ function arr:function-item-invalid() {
 
 declare
     %test:assertError("FOAY0001")
-function arr:get-item-out-of-bounds() {
+function arr:get-item-out-of-bounds-positive() {
     let $a := [13, 10, 14]
     return
         array:get($a, 22)
```

Comment:
I'd prefer to test cornercase with value 4 (that is maybe related to your int -1 change) in stead of 22 of which that should fail anyway. for "4" we might find an/the implementation issue.